### PR TITLE
make vep_struct_to_csq compatible with vep 101

### PR DESCRIPTION
**Pull request by klaricch**
I ran into errors trying to use vep_struct_to_csq on a file that used v101 for vep annotations because the code references some fields not found in vep101. This PR edits the function to avoid errors when using vep v101.

### CODE REVIEW

**Comment by KoalaQin** [comment]
I will wait until the checks to pass first.

File: gnomad/utils/vep.py [hunk 1]
```diff
@@ -478,32 +478,43 @@ def get_csq_from_struct(
...
+                ),
+                "mirna": hl.delimit(element.mirna, "&") if "mirna" in element else None,
+                "protein_position": hl.str(element.protein_start) + hl.if_else(
+                    element.protein_start == element.protein_end,
+                    "",
+                    "-" + hl.str(element.protein_end),
+                ),
+                "uniprot_isoform": (
+                    hl.delimit(element.uniprot_isoform, "&")
+                    if "uniprot_isoform" in element
```
Comment (KoalaQin) [inline gnomad/utils/vep.py:505]: How about those all-missing fields? Like `swissprot, trembl, uniparc`? And the new fields: `mane_select, mane_plus_clinical, mirna`?

Comment (KoalaQin) [reply gnomad/utils/vep.py:505]: I know `uniprot_isoform` is new.

Comment (klaricch) [reply gnomad/utils/vep.py:505]: i don't think the other fields are impacted

**Review by KoalaQin — APPROVED** [review]
LGTM! 
I tried this on DRD2 of v4 genomes, the older version of the function didn't run into errors, I guess this is only for a larger dataset. 
It worked fine with your version too.

**Pull request merged**